src/status_im/chat/models.cljs

```diff
@@ -107,6 +107,25 @@
               (when (and public? new?)
                 (transport.filters/load-chat chat-id)))))
 
+(fx/defn ensure-chats
+  "Add chats to db and update"
+  [{:keys [db] :as cofx} chats]
+  (let [chats (map #(merge
```

Comment:
can we use anonymous function instead of fn literal? It is a bit more readable when body contains more than one form